Dockerfile

```diff
@@ -1,31 +1,54 @@
-FROM public.ecr.aws/p9i6h6j0/aws-pdk:latest as build
+FROM --platform=linux/x86_64 public.ecr.aws/amazonlinux/amazonlinux:latest as build
```

Comment:
Pls remove the --platform=linux/x86_64 as it may cause some issues in the cdk build if the users use cdk build. 
Instead, add a comment below to instruct Mac users to specify the platform during docker build. For instance, `docker build --platform=linux/x86_64 -t threat-composer-docker .`